TestHelpers.Tests/SkipOnUnixAttribute.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+
+namespace System.IO.Abstractions.TestingHelpers.Tests
+{
+    internal static class SkipReason
```

Comment:
I'd probably put this in its own class. You?

Reply:
Yeah, will do